projects/aca-content/folder-rules/src/rule-details/conditions/rule-composite-condition.ui-component.scss

```diff
@@ -40,6 +40,10 @@
       & > :nth-child(2) {
         flex: 1;
       }
+
+      mat-form-field {
```

Comment:
is it possible not to use 'mat' selector?

Reply:
done